### DISCUSSION TITLE
Mac 打包的程序无法打开跳转链接

### DISCUSSION THREAD

**Discussion by kayshen0817** [tw93/Pake#621] [Q&A]
请问，我用小白的方法，fork 之后打包的程序。请问为什么打包的mac 程序都无法跳转或打开新链接呢，比如飞书多维表和知识星球之类的，点开新的链接就是没有反应，既不跳转也不打开新页面来打开新的链接。就是没有任何反应。请问是我哪里打包出错了吗？windows的包没有这个问题， MultiArch 选项也选上了的。

我的机器是:MacBook Pro (16-inch, 2019)，CPU 是英特尔的，系统是 12.6.5。 Safari 是 15.6.1 的。

请问有哪位大佬能指点一下吗？

**Comment by tw93** [comment]
看起来你需要试试调试模式，我记得飞书的跳转逻辑写得很反人类
